src/components/HeaderLink/HeaderLink.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC } from 'react';
+import { NavLink } from 'react-router-dom';
+
+import styles from './HeaderLink.module.scss';
+
+interface LinkProps {
+  path: string;
+  name: string;
+}
+
+export const HeaderLink: FC<LinkProps> = ({ path, name }) => {
```

Comment:
```suggestion
export const HeaderLink: FC<Props> = ({ path, name }) => {
```

Нейминг скачет. HeaderLink, но LinkProps. Я бы вообще посоветовал, если типы пропсов не торчат наружу, то можно называть просто `Props `и ок.